chapter_linear-networks/image-classification-dataset.md

```diff
@@ -67,7 +67,7 @@ mnist_train, mnist_test = tf.keras.datasets.fashion_mnist.load_data()
 ```
 
 Fashion-MNIST consists of images from 10 categories, each represented
-by 6000 images in the training dataset and by 1000 in the test dataset.
+by 60000 images in the training dataset and by 10000 in the test dataset.
```

Comment:
These are already correct since we have 6000 examples for **EACH** class. Please revert these changes.